### DISCUSSION TITLE
Get all data past 90 day access

### DISCUSSION THREAD

**Discussion by dvnrsn** [rusq/slackdump#247] [Q&A] [answered]
I was able to get a lot of public channel data using mattermost [migrate from slack](https://docs.mattermost.com/onboard/migrating-to-mattermost.html#migrate-from-slack) and it's my understanding that if I want DM data especially beyond 90 days I will need to upgrade slack plan (currently free tier). I wanted to check if this understanding is correct.

I'm also missing some images and files prior to 90 days. Seems like slack is being stringent.

**Comment by dvnrsn** [comment]
https://slack.com/help/articles/204897248-Guide-to-Slack-import-and-export-tools
<img width="669" alt="image" src="https://github.com/rusq/slackdump/assets/9112811/d923184a-2b36-4e7f-9b2e-a27aed567271">

**Comment by rusq** [comment]
Hey @dvnrsn , I can only assume that it is so, never tried myself.

**Reply by rusq** [reply, marked as answer]
So, the obvious solution is to ban everyone, purchase the non-free plan, run the export.  That is if Slack has not deleted everything beyond a year yet.
